### DISCUSSION TITLE
.Net Standard ContentDialog IDisposable

### DISCUSSION THREAD

**Discussion by HavenDV** [unoplatform/uno#9563] [Ideas]
Maybe add an empty IDisposable.Dispose() for ContentDialog for .Net Standard? It's quite tricky in library code to check if Dispose needs to be called for a specific platform

**Comment by jeromelaban** [comment]
None of the WinUI `UIElement` types are disposable. Why do you need to call `Dispose()` ?

**Reply by HavenDV** [reply]
What I mean is that the ContentDialog for mobile platforms shows a warning that a Dispose() call was missed. If it's meant to be required by mobile platforms, maybe add it to the .net standard as well so that users don't miss this point?

**Reply by jeromelaban** [reply]
I see, this is not something we can change, as it's coming from the native iOS/Android implementations, and it's actually counter productive (and destabilizing for the runtime) to invoke the `Dispose` method.

**Reply by HavenDV** [reply]
Perhaps it's worth specifying this with Dispose override/new and the Obsolete attribute?
